### ISSUE TITLE
Is this repo dead?

### ISSUE THREAD

**Issue by megaadam** [grafov/m3u8#184]
Is this repo dead?
No PR responses and... A lot of the DeepSource-PR failures are are red because of stuff already on master.

**Comment by bwildenhain** [comment]
Yes, it is obviously dead: There used to be around four releases per year until 2019 and no interaction from grafov since that year. There are some forks which are actively maintained, e.g.:

- https://github.com/edgeware/m3u8
- https://github.com/zencoder/m3u8

As they don't have public issue trackers, I don't know how interested they are in collaboration with external users.

**Comment by 89z** [comment]
I have an alternate package here:

https://github.com/89z/format

**Comment by grafov** [comment]
Well. I have a little time for the project. The code is still used in a couple of proprietary projects. May be it is dangerous due to issues of the project but the proprietary code ...ok you know )

The patches and maintaining for the project are welcome. My email grafov@gmail.com or you can reach me in the issues. Unfortunataly my professional activity moved from video streaming several years ago. But I continue to work with Go language.

I think the best way it is move the project under umbrella of some non profit organization who interested in Go libraries. Or add new maintainers who interest in this code in their projects. There was  persons who maintain the project after me. I appreciated their help very much. But any of us sometime change their interests or professional area so last years the project abandoned.

Ok, I'll try to keep the code anyway ) If you have actively maintainad replacements I'll put the links to README.

**Comment by hr8** [comment]
If it helps, I have an HLS package that I keep up to date:

https://github.com/hr8/rosso

**Comment by grafov** [comment]
> https://github.com/hr8/rosso

Thank you added to readme.
